_docs/tickets/snippets.md

```diff
@@ -56,4 +56,25 @@ Reference #: <b>CRB-01092-002</b>
 Subject: <b>Do you accept purchase orders?</b>
 
 We'll be in contact shortly.
-{% endhighlight %}
\ No newline at end of file
+{% endhighlight %}
+
+## Prompts
+
+Snippets also support prompts for information prior to their pasting. You can use these prompts to dynamically insert relevant values into the snippet or to be used in further scripting. You set up prompts in the "Prompts"
+section at the bottom of the snippet editor.
```

Comment:
Let phpStorm soft wrap the long lines. We don't need to hard wrap them unless they're in the YAML frontmatter at the top (like `summary:`).